messaging/Controllers/BundlesController.cs

```diff
@@ -75,6 +76,19 @@ public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdict
                 return BadRequest("Pagination does not support specifying a page without a _since parameter");
             }
 
+            if (certificateNumber != null && deathYear != null) {

```

Comment:
This made me think of a couple questions - should this be `certificateNumber == null || deathYear == null` then use the default GetOutgoingMessageItems? And if they only sent certificateNumber or deathYear then should we return an error saying they need to provide all business identifiers to select by business ids? I.e. you'd need to implement 3 cases
1. No business id provided, use default behavior
2. One business id provided, return error saying you need all business ids
3. All business ids provided, return message history for business ids

Reply:
Just thought of one more this for case 2. This would definitely require coordination with the STEVE team so they could return an error in the same situation. So maybe its best to just have the 2 cases and work with what the user provides.